src/components/CustomSelectDropdown/account-dropdown/AccountDropdown/__tests__/AccountDropdown.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import AccountDropdown from '..';
+import useAuthContext from '@site/src/hooks/useAuthContext';
+import userEvent from '@testing-library/user-event';
+import AuthProvider from '@site/src/contexts/auth/auth.provider';
+import { render } from '@testing-library/react';
+
+jest.mock('@site/src/hooks/useAuthContext');
+
+const fake_accounts = [
```

Comment:
add the type for this array, in the future if the account type is changed this will fail and we'll get better feed back from our test.

Reply:
Done, updated it also in some of your tests where it was missing